### PR TITLE
Add .NET library

### PR DESCRIPTION
**Pull request by dariogriffo**
I have created a new .NET library implementation. Under development but published as alpha on [Nuget](https://www.nuget.org/packages/Bluesky.Net/#readme-body-tab)

### CODE REVIEW

**Comment by dgellow** [comment]
Thanks @dariogriffo, I want to play around with a desktop GUI, your library will likely make my life easier. Just curious, how far did you go? What is currently missing?

**Comment by dariogriffo** [comment]
> Thanks @dariogriffo, I want to play around with a desktop GUI, your library will likely make my life easier. Just curious, how far did you go? What is currently missing?

I'm planning to have create post + a query to the user feed before the end of this week. Then I'll be off for almost 2 weeks on holidays and I have no intention to open my laptop :smile:

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ List of projects and implementations in the AT protocol ecosystem. Many are WIP,
 - [atproto](https://github.com/myConsciousness/atproto.dart/tree/main/packages/atproto) and [bluesky](https://github.com/myConsciousness/atproto.dart/tree/main/packages/bluesky) (Dart)
 - [bskyrb](https://github.com/ShreyanJain9/bskyrb) (Ruby): new, not stable
 - [blue-pyinthe-sky](https://github.com/robcerda/blue-pyinthe-sky) (Python) new, not stable. Packaging it up into an installable module is a work in progress. 
-
+- [BlueSky.Net](https://github.com/dariogriffo/bluesky-net) (.NET) new, WIP. Available to download in [Nuget](https://www.nuget.org/packages/Bluesky.Net/)
 ## Clients
```
Comment (emilyliu7321) [inline README.md:26]:
```suggestion
- [BlueSky.Net](https://github.com/dariogriffo/bluesky-net) (.NET) new, WIP. Available to download in [Nuget](https://www.nuget.org/packages/Bluesky.Net/)

## Clients
```

**Comment by emilyliu7321** [comment]
thanks! merging this PR :) 

we'll be deprecating this repo shortly and using https://atproto.com/community/projects as the public-facing page for community projects. you can also make PRs to that site here: https://github.com/bluesky-social/atproto-website

**Pull request merged**